Cargo.toml

```diff
@@ -11,10 +11,14 @@ repository = "https://github.com/PolyMeilex/rfd"
 documentation = "https://docs.rs/rfd"
 
 [features]
-default = ["gtk3"]
+default = ["gtk3", "async-std"]
```

Comment:
In this case "async-std"  is a xdg-portal specific dep, so is there any reason to include it in gtk3 default?
You have to do `default-features = false` in order to enable xdg-portal backend, so this default will never be used.

Reply:
I didn't realize `gtk3` and `xdg-portal` were mutually exclusive. Removed.

Reply:
Yep, GTK3 does support xdg portal internally